### DISCUSSION TITLE
New devices support, tests, feedback

### DISCUSSION THREAD

**Discussion by xyzroe** [xyzroe/XZG#76] [Q&A]
Hi guys, @codm @tube0013 
I have added support for your gateways to this firmware.
I would be grateful for tests and feedback

P.S. There have also been a number of updates recently, and it seems to have improved stability.

**Comment by el-bakkali** [comment]
Ive tried flashing my  ZB-GW03 with web flashed and 20240914
 https://templates.blakadder.com/ewelink_ZB-GW03.html

it comes with ESP32-D0WD-V3 with CC2652P

however it doesnt work
i had to revert back to flashing serial to Tasmota32 zigbeebridge https://jason2866.github.io/Tasmota-specials/

I really want to ditch tasmota in favour of this xgz firmware
if anyone knows how to get it working for our device let me know
<img width="495" alt="image" src="https://github.com/user-attachments/assets/24d9ab92-11c4-4747-91f5-5dcf83b52492">

**Reply by xyzroe** [reply]
ZB-GW03-V1.2 comes with ERF32, not 2652. 

ERF still not supported

**Reply by el-bakkali** [reply]
Thanks for confirming 
I guess this uses the new ember driver firmware 

is there any way I can contribute to testing to bring device support to ERF32

**Reply by xyzroe** [reply]
I don't have enough motivation and time for XZG, sorry. 
Maybe someone will make this. Or something will change in my life.
